eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Utxo.hs

```diff
@@ -124,8 +129,9 @@ data UtxoEnv era
   = UtxoEnv
       SlotNo
       (Core.PParams era)
-      (Map (KeyHash 'StakePool (Crypto era)) (PoolParams (Crypto era)))
+      (Map (KeyHash 'StakePool (Crypto era)) (PoolParams (Crypto era))) -- DO WE NEED THIS, IN DPSTATE
```

Comment:
We do need the pool parameters in the UTXO environment, so that we can recognize which stake pool registration certificates are new and which are re-registrations (only the former pay a deposit).

Reply:
I see, we will leave it.